README.md

```diff
@@ -1,8 +1,4 @@
 # Generator for MicroProfile Rest Client
-
-[![Marketplace Version](https://vsmarketplacebadge.apphb.com/version/MicroProfile-Community.mp-rest-client-generator-vscode-ext.svg "Current Release")](https://marketplace.visualstudio.com/items?itemName=MicroProfile-Community.mp-rest-client-generator-vscode-ext)
```

Comment:
I think we should keep the link to the marketplace

Reply:
Unfortunately, svg is not permitted anymore, unless someone told me how upload png files to the website

Reply:
I don't know if the image matters, we can probably remove it. But we should not remove the marketplace link altogether.

Reply:
hi @TrevCraw

I did it by following what the LT does.

Reply:
LGTM